src/components/Inputs/InputCheckbox.js

```diff
@@ -0,0 +1,39 @@
+export const InputCheckbox = props => {
+      const {options, value  } = props
+      const changeFeatureHandler = e => {
+      const value = e.target.value;
+      const isChecked = e.target.checked;
+  
+      if (isChecked) {
+        const newValue = [...props.value, value];
+        props.onChange(newValue);
+      } else {
+        const newValue = props.value.filter(x => x !== value);
+        props.onChange(newValue);
+      }
+    }
+  
+    return (
+      <div className="form-group">
+        {props.options.map(option => (

```

Comment:
Change to just "options". The same for props.value and props in other components where you use a desctructure.